.travis.yml

```diff
@@ -11,6 +11,7 @@ install:
 - pip install -r requirements.txt
 - pip install coveralls readme_renderer
 script:
+- python setup.py install
```

Comment:
This was added to invoke the  Popen(['pyethapp']). Otherwise it won't find the executable.